2018-10-26 15:49:07,273 ERROR 35536 [-/::1/-/97ms POST /addClass] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'undefined' in 'field list'
    at Query.Sequence._packetToError (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Protocol.js:278:23)
    at Parser.write (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Connection.js:91:28)
    at Socket.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Connection.js:502:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    --------------------
    at Pool.query (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\lib\client.js:34:20)
    at RDSClient.proto.query (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\linge l yu\Desktop\egg-example\node_modules\co\index.js:65:19)
    at C:\Users\linge l yu\Desktop\egg-example\node_modules\co\index.js:54:5
    sql: 
            insert into classificationfirst(name) values(undefined)
        
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'undefined' in 'field list'"
sqlState: "42S22"
index: 0
sql: "\n            insert into classificationfirst(name) values(undefined)\n        "
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true"}
name: "ER_BAD_FIELD_ERRORError"
pid: 35536
hostname: LYU129PRI2

2018-10-26 16:15:59,291 ERROR 35536 [-/::1/-/8ms POST /addClass?firstClassName=123&secClassName=21] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'undefined' in 'field list'
    at Query.Sequence._packetToError (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Protocol.js:278:23)
    at Parser.write (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Connection.js:91:28)
    at Socket.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Connection.js:502:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    --------------------
    at Pool.query (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\lib\client.js:34:20)
    at RDSClient.proto.query (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\linge l yu\Desktop\egg-example\node_modules\co\index.js:65:19)
    at C:\Users\linge l yu\Desktop\egg-example\node_modules\co\index.js:54:5
    sql: 
            insert into classificationfirst(name) values(undefined)
        
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'undefined' in 'field list'"
sqlState: "42S22"
index: 0
sql: "\n            insert into classificationfirst(name) values(undefined)\n        "
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true"}
name: "ER_BAD_FIELD_ERRORError"
pid: 35536
hostname: LYU129PRI2

2018-10-26 16:16:40,646 ERROR 35536 [-/::1/-/5ms POST /addClass] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'undefined' in 'field list'
    at Query.Sequence._packetToError (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Protocol.js:278:23)
    at Parser.write (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Connection.js:91:28)
    at Socket.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Connection.js:502:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    --------------------
    at Pool.query (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\lib\client.js:34:20)
    at RDSClient.proto.query (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\linge l yu\Desktop\egg-example\node_modules\co\index.js:65:19)
    at C:\Users\linge l yu\Desktop\egg-example\node_modules\co\index.js:54:5
    sql: 
            insert into classificationfirst(name) values(undefined)
        
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'undefined' in 'field list'"
sqlState: "42S22"
index: 0
sql: "\n            insert into classificationfirst(name) values(undefined)\n        "
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true"}
name: "ER_BAD_FIELD_ERRORError"
pid: 35536
hostname: LYU129PRI2

2018-10-26 16:18:19,178 ERROR 35536 [-/::1/-/8ms POST /addClass] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'undefined' in 'field list'
    at Query.Sequence._packetToError (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Protocol.js:278:23)
    at Parser.write (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Connection.js:91:28)
    at Socket.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Connection.js:502:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    --------------------
    at Pool.query (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\lib\client.js:34:20)
    at RDSClient.proto.query (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\linge l yu\Desktop\egg-example\node_modules\co\index.js:65:19)
    at C:\Users\linge l yu\Desktop\egg-example\node_modules\co\index.js:54:5
    sql: 
            insert into classificationfirst(name) values(undefined)
        
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'undefined' in 'field list'"
sqlState: "42S22"
index: 0
sql: "\n            insert into classificationfirst(name) values(undefined)\n        "
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true"}
name: "ER_BAD_FIELD_ERRORError"
pid: 35536
hostname: LYU129PRI2

2018-10-26 16:18:48,159 ERROR 35536 [-/::1/-/5ms POST /addClass] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'undefined' in 'field list'
    at Query.Sequence._packetToError (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Protocol.js:278:23)
    at Parser.write (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Connection.js:91:28)
    at Socket.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Connection.js:502:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    --------------------
    at Pool.query (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\lib\client.js:34:20)
    at RDSClient.proto.query (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\linge l yu\Desktop\egg-example\node_modules\co\index.js:65:19)
    at C:\Users\linge l yu\Desktop\egg-example\node_modules\co\index.js:54:5
    sql: 
            insert into classificationfirst(name) values(undefined)
        
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'undefined' in 'field list'"
sqlState: "42S22"
index: 0
sql: "\n            insert into classificationfirst(name) values(undefined)\n        "
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true"}
name: "ER_BAD_FIELD_ERRORError"
pid: 35536
hostname: LYU129PRI2

2018-10-26 16:19:24,639 ERROR 35536 [-/::1/-/7ms POST /addClass] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'undefined' in 'field list'
    at Query.Sequence._packetToError (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Protocol.js:278:23)
    at Parser.write (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Connection.js:91:28)
    at Socket.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Connection.js:502:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    --------------------
    at Pool.query (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\lib\client.js:34:20)
    at RDSClient.proto.query (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\linge l yu\Desktop\egg-example\node_modules\co\index.js:65:19)
    at C:\Users\linge l yu\Desktop\egg-example\node_modules\co\index.js:54:5
    sql: 
            insert into classificationfirst(name) values(undefined)
        
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'undefined' in 'field list'"
sqlState: "42S22"
index: 0
sql: "\n            insert into classificationfirst(name) values(undefined)\n        "
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true"}
name: "ER_BAD_FIELD_ERRORError"
pid: 35536
hostname: LYU129PRI2

2018-10-26 16:19:44,641 ERROR 35536 [-/::1/-/6ms POST /addClass] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'undefined' in 'field list'
    at Query.Sequence._packetToError (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Protocol.js:278:23)
    at Parser.write (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Connection.js:91:28)
    at Socket.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Connection.js:502:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    --------------------
    at Pool.query (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\lib\client.js:34:20)
    at RDSClient.proto.query (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\linge l yu\Desktop\egg-example\node_modules\co\index.js:65:19)
    at C:\Users\linge l yu\Desktop\egg-example\node_modules\co\index.js:54:5
    sql: 
            insert into classificationfirst(name) values(undefined)
        
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'undefined' in 'field list'"
sqlState: "42S22"
index: 0
sql: "\n            insert into classificationfirst(name) values(undefined)\n        "
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true"}
name: "ER_BAD_FIELD_ERRORError"
pid: 35536
hostname: LYU129PRI2

2018-10-26 16:20:47,273 ERROR 35536 [-/::1/-/6ms POST /addClass] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'undefined' in 'field list'
    at Query.Sequence._packetToError (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Protocol.js:278:23)
    at Parser.write (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Connection.js:91:28)
    at Socket.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Connection.js:502:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    --------------------
    at Pool.query (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\lib\client.js:34:20)
    at RDSClient.proto.query (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\linge l yu\Desktop\egg-example\node_modules\co\index.js:65:19)
    at C:\Users\linge l yu\Desktop\egg-example\node_modules\co\index.js:54:5
    sql: 
            insert into classificationfirst(name) values(undefined)
        
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'undefined' in 'field list'"
sqlState: "42S22"
index: 0
sql: "\n            insert into classificationfirst(name) values(undefined)\n        "
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true"}
name: "ER_BAD_FIELD_ERRORError"
pid: 35536
hostname: LYU129PRI2

2018-10-26 16:20:54,954 ERROR 35536 [-/::1/-/8ms POST /addClass] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'undefined' in 'field list'
    at Query.Sequence._packetToError (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Protocol.js:278:23)
    at Parser.write (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Connection.js:91:28)
    at Socket.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Connection.js:502:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    --------------------
    at Pool.query (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\lib\client.js:34:20)
    at RDSClient.proto.query (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\linge l yu\Desktop\egg-example\node_modules\co\index.js:65:19)
    at C:\Users\linge l yu\Desktop\egg-example\node_modules\co\index.js:54:5
    sql: 
            insert into classificationfirst(name) values(undefined)
        
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'undefined' in 'field list'"
sqlState: "42S22"
index: 0
sql: "\n            insert into classificationfirst(name) values(undefined)\n        "
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true"}
name: "ER_BAD_FIELD_ERRORError"
pid: 35536
hostname: LYU129PRI2

2018-10-26 16:21:08,433 ERROR 35536 [-/::1/-/6ms POST /addClass] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'undefined' in 'field list'
    at Query.Sequence._packetToError (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Protocol.js:278:23)
    at Parser.write (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Connection.js:91:28)
    at Socket.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Connection.js:502:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    --------------------
    at Pool.query (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\lib\client.js:34:20)
    at RDSClient.proto.query (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\linge l yu\Desktop\egg-example\node_modules\co\index.js:65:19)
    at C:\Users\linge l yu\Desktop\egg-example\node_modules\co\index.js:54:5
    sql: 
            insert into classificationfirst(name) values(undefined)
        
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'undefined' in 'field list'"
sqlState: "42S22"
index: 0
sql: "\n            insert into classificationfirst(name) values(undefined)\n        "
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true"}
name: "ER_BAD_FIELD_ERRORError"
pid: 35536
hostname: LYU129PRI2

2018-10-26 16:21:20,011 ERROR 35536 [-/::1/-/7ms POST /addClass] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'undefined' in 'field list'
    at Query.Sequence._packetToError (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Protocol.js:278:23)
    at Parser.write (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Connection.js:91:28)
    at Socket.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Connection.js:502:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    --------------------
    at Pool.query (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\lib\client.js:34:20)
    at RDSClient.proto.query (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\linge l yu\Desktop\egg-example\node_modules\co\index.js:65:19)
    at C:\Users\linge l yu\Desktop\egg-example\node_modules\co\index.js:54:5
    sql: 
            insert into classificationfirst(name) values(undefined)
        
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'undefined' in 'field list'"
sqlState: "42S22"
index: 0
sql: "\n            insert into classificationfirst(name) values(undefined)\n        "
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true"}
name: "ER_BAD_FIELD_ERRORError"
pid: 35536
hostname: LYU129PRI2

2018-10-26 16:23:43,076 ERROR 35536 [-/::1/-/12ms POST /addClass] nodejs.ER_DUP_ENTRYError: ER_DUP_ENTRY: Duplicate entry '123' for key 'onlyone'
    at Query.Sequence._packetToError (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Protocol.js:278:23)
    at Parser.write (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Connection.js:91:28)
    at Socket.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Connection.js:502:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    --------------------
    at Pool.query (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\lib\client.js:34:20)
    at RDSClient.proto.query (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\linge l yu\Desktop\egg-example\node_modules\co\index.js:65:19)
    at C:\Users\linge l yu\Desktop\egg-example\node_modules\co\index.js:54:5
    sql: 
            insert into classificationfirst(name) values(123)
        
code: "ER_DUP_ENTRY"
errno: 1062
sqlMessage: "Duplicate entry '123' for key 'onlyone'"
sqlState: "23000"
index: 0
sql: "\n            insert into classificationfirst(name) values(123)\n        "
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true"}
name: "ER_DUP_ENTRYError"
pid: 35536
hostname: LYU129PRI2

2018-10-26 16:24:45,170 ERROR 564 nodejs.unhandledExceptionError: read ECONNRESET
    at _errnoException (util.js:992:11)
    at TCP.onread (net.js:618:25)
code: "ECONNRESET"
errno: "ECONNRESET"
syscall: "read"
name: "unhandledExceptionError"
pid: 564
hostname: LYU129PRI2

2018-10-26 16:28:03,369 ERROR 35896 [-/::1/-/70ms POST /addClass] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column '电脑' in 'field list'
    at Query.Sequence._packetToError (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Protocol.js:278:23)
    at Parser.write (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Connection.js:91:28)
    at Socket.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Connection.js:502:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    --------------------
    at Pool.query (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\lib\client.js:34:20)
    at RDSClient.proto.query (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\linge l yu\Desktop\egg-example\node_modules\co\index.js:65:19)
    at C:\Users\linge l yu\Desktop\egg-example\node_modules\co\index.js:54:5
    sql: 
            insert into classificationfirst(name) values(电脑)
        
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column '电脑' in 'field list'"
sqlState: "42S22"
index: 0
sql: "\n            insert into classificationfirst(name) values(电脑)\n        "
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true"}
name: "ER_BAD_FIELD_ERRORError"
pid: 35896
hostname: LYU129PRI2

2018-10-26 16:29:02,245 ERROR 30596 [-/::1/-/66ms POST /addClass] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column '电脑' in 'field list'
    at Query.Sequence._packetToError (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Protocol.js:278:23)
    at Parser.write (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Connection.js:91:28)
    at Socket.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Connection.js:502:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    --------------------
    at Pool.query (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\lib\client.js:34:20)
    at RDSClient.proto.query (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\linge l yu\Desktop\egg-example\node_modules\co\index.js:65:19)
    at C:\Users\linge l yu\Desktop\egg-example\node_modules\co\index.js:54:5
    sql: 
            insert into classificationfirst(name) values(电脑)
        
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column '电脑' in 'field list'"
sqlState: "42S22"
index: 0
sql: "\n            insert into classificationfirst(name) values(电脑)\n        "
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true"}
name: "ER_BAD_FIELD_ERRORError"
pid: 30596
hostname: LYU129PRI2

2018-10-26 16:34:16,197 ERROR 23440 [-/::1/-/81ms POST /addClass] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column '笔记本' in 'field list'
    at Query.Sequence._packetToError (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Protocol.js:278:23)
    at Parser.write (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Connection.js:91:28)
    at Socket.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Connection.js:502:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    --------------------
    at Pool.query (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\lib\client.js:34:20)
    at RDSClient.proto.query (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\linge l yu\Desktop\egg-example\node_modules\co\index.js:65:19)
    at C:\Users\linge l yu\Desktop\egg-example\node_modules\co\index.js:54:5
    sql: 
            insert into classificationfirst(name) values(笔记本)
        
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column '笔记本' in 'field list'"
sqlState: "42S22"
index: 0
sql: "\n            insert into classificationfirst(name) values(笔记本)\n        "
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true"}
name: "ER_BAD_FIELD_ERRORError"
pid: 23440
hostname: LYU129PRI2

2018-10-26 16:37:50,663 ERROR 25228 [-/::1/-/69ms POST /addClass] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column '笔记本1' in 'field list'
    at Query.Sequence._packetToError (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Protocol.js:278:23)
    at Parser.write (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Connection.js:91:28)
    at Socket.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Connection.js:502:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    --------------------
    at Pool.query (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\lib\client.js:34:20)
    at RDSClient.proto.query (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\linge l yu\Desktop\egg-example\node_modules\co\index.js:65:19)
    at C:\Users\linge l yu\Desktop\egg-example\node_modules\co\index.js:54:5
    sql: 
            insert into classificationfirst(name) values(笔记本1)
        
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column '笔记本1' in 'field list'"
sqlState: "42S22"
index: 0
sql: "\n            insert into classificationfirst(name) values(笔记本1)\n        "
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true"}
name: "ER_BAD_FIELD_ERRORError"
pid: 25228
hostname: LYU129PRI2

2018-10-26 16:37:57,902 ERROR 25228 [-/::1/-/6ms POST /addClass] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column '笔记本1' in 'field list'
    at Query.Sequence._packetToError (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Protocol.js:278:23)
    at Parser.write (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Connection.js:91:28)
    at Socket.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Connection.js:502:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    --------------------
    at Pool.query (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\lib\client.js:34:20)
    at RDSClient.proto.query (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\linge l yu\Desktop\egg-example\node_modules\co\index.js:65:19)
    at C:\Users\linge l yu\Desktop\egg-example\node_modules\co\index.js:54:5
    sql: 
            insert into classificationfirst(name) values(笔记本1)
        
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column '笔记本1' in 'field list'"
sqlState: "42S22"
index: 0
sql: "\n            insert into classificationfirst(name) values(笔记本1)\n        "
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true"}
name: "ER_BAD_FIELD_ERRORError"
pid: 25228
hostname: LYU129PRI2

2018-10-26 16:38:44,299 ERROR 25228 [-/::1/-/11ms POST /addClass] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column '笔记本12123' in 'field list'
    at Query.Sequence._packetToError (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Protocol.js:278:23)
    at Parser.write (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Connection.js:91:28)
    at Socket.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Connection.js:502:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    --------------------
    at Pool.query (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\lib\client.js:34:20)
    at RDSClient.proto.query (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\linge l yu\Desktop\egg-example\node_modules\co\index.js:65:19)
    at C:\Users\linge l yu\Desktop\egg-example\node_modules\co\index.js:54:5
    sql: 
            insert into classificationfirst(name) values(笔记本12123)
        
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column '笔记本12123' in 'field list'"
sqlState: "42S22"
index: 0
sql: "\n            insert into classificationfirst(name) values(笔记本12123)\n        "
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true"}
name: "ER_BAD_FIELD_ERRORError"
pid: 25228
hostname: LYU129PRI2

2018-10-26 16:39:57,722 ERROR 30812 [-/::1/-/72ms POST /addClass] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column '123sasa' in 'field list'
    at Query.Sequence._packetToError (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Protocol.js:278:23)
    at Parser.write (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Connection.js:91:28)
    at Socket.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Connection.js:502:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    --------------------
    at Pool.query (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\lib\client.js:34:20)
    at RDSClient.proto.query (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\linge l yu\Desktop\egg-example\node_modules\co\index.js:65:19)
    at C:\Users\linge l yu\Desktop\egg-example\node_modules\co\index.js:54:5
    sql: 
            insert into classificationfirst(name) values(123sasa)
        
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column '123sasa' in 'field list'"
sqlState: "42S22"
index: 0
sql: "\n            insert into classificationfirst(name) values(123sasa)\n        "
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true"}
name: "ER_BAD_FIELD_ERRORError"
pid: 30812
hostname: LYU129PRI2

2018-10-26 16:46:36,288 ERROR 30840 [-/::1/-/80ms POST /addClass] nodejs.TypeError: Cannot read property 'id' of undefined
    at Classification.addClass (C:\Users\linge l yu\Desktop\egg-example\app\service\classification.js:39:140)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true"}
pid: 30840
hostname: LYU129PRI2

2018-10-26 16:47:16,695 ERROR 26492 [-/::1/-/68ms POST /addClass] nodejs.ER_DUP_ENTRYError: ER_DUP_ENTRY: Duplicate entry '文具' for key 'onlyone'
    at Query.Sequence._packetToError (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Protocol.js:278:23)
    at Parser.write (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Connection.js:91:28)
    at Socket.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Connection.js:502:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    --------------------
    at Pool.query (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\lib\client.js:34:20)
    at RDSClient.proto.query (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\linge l yu\Desktop\egg-example\node_modules\co\index.js:65:19)
    at C:\Users\linge l yu\Desktop\egg-example\node_modules\co\index.js:54:5
    sql: 
            insert into classificationfirst(name) values('文具')
        
code: "ER_DUP_ENTRY"
errno: 1062
sqlMessage: "Duplicate entry '文具' for key 'onlyone'"
sqlState: "23000"
index: 0
sql: "\n            insert into classificationfirst(name) values('文具')\n        "
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true"}
name: "ER_DUP_ENTRYError"
pid: 26492
hostname: LYU129PRI2

2018-10-26 16:48:03,997 ERROR 26492 [-/::1/-/6ms POST /addClass] nodejs.ER_DUP_ENTRYError: ER_DUP_ENTRY: Duplicate entry '文具' for key 'onlyone'
    at Query.Sequence._packetToError (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Protocol.js:278:23)
    at Parser.write (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Connection.js:91:28)
    at Socket.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Connection.js:502:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    --------------------
    at Pool.query (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\lib\client.js:34:20)
    at RDSClient.proto.query (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\linge l yu\Desktop\egg-example\node_modules\co\index.js:65:19)
    at C:\Users\linge l yu\Desktop\egg-example\node_modules\co\index.js:54:5
    sql: 
            insert into classificationfirst(name) values('文具')
        
code: "ER_DUP_ENTRY"
errno: 1062
sqlMessage: "Duplicate entry '文具' for key 'onlyone'"
sqlState: "23000"
index: 0
sql: "\n            insert into classificationfirst(name) values('文具')\n        "
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true"}
name: "ER_DUP_ENTRYError"
pid: 26492
hostname: LYU129PRI2

2018-10-26 16:57:20,237 ERROR 19068 [-/::1/-/17ms GET /getClass] nodejs.ER_PARSE_ERRORError: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'desc cfs.firstClassName' at line 4
    at Query.Sequence._packetToError (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Protocol.js:278:23)
    at Parser.write (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Connection.js:91:28)
    at Socket.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Connection.js:502:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    --------------------
    at Pool.query (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\lib\client.js:34:20)
    at RDSClient.proto.query (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\linge l yu\Desktop\egg-example\node_modules\co\index.js:65:19)
    at C:\Users\linge l yu\Desktop\egg-example\node_modules\co\index.js:54:5
    sql: 
        SELECT
            cfs.*,
            cf.NAME firstClassName,
            cs.NAME secClassName 
        FROM
            class_first_sec cfs
            LEFT JOIN classificationfirst cf ON cfs.classFirstId = cf.id
            LEFT JOIN classificationsec cs ON cfs.classSecId = cs.id order by desc cfs.firstClassName
code: "ER_PARSE_ERROR"
errno: 1064
sqlMessage: "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'desc cfs.firstClassName' at line 4"
sqlState: "42000"
index: 0
sql: "\n        SELECT\n            cfs.*,\n            cf.NAME firstClassName,\n            cs.NAME secClassName \n        FROM\n            class_first_sec cfs\n            LEFT JOIN classificationfirst cf ON cfs.classFirstId = cf.id\n            LEFT JOIN classificationsec cs ON cfs.classSecId = cs.id order by desc cfs.firstClassName"
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true"}
name: "ER_PARSE_ERRORError"
pid: 19068
hostname: LYU129PRI2

2018-10-26 17:00:22,427 ERROR 19068 [-/::1/-/18ms GET /getClass] nodejs.ER_PARSE_ERRORError: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'desc cfs.firstClassName' at line 4
    at Query.Sequence._packetToError (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Protocol.js:278:23)
    at Parser.write (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Connection.js:91:28)
    at Socket.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Connection.js:502:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    --------------------
    at Pool.query (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\lib\client.js:34:20)
    at RDSClient.proto.query (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\linge l yu\Desktop\egg-example\node_modules\co\index.js:65:19)
    at C:\Users\linge l yu\Desktop\egg-example\node_modules\co\index.js:54:5
    sql: 
        SELECT
            cfs.*,
            cf.NAME firstClassName,
            cs.NAME secClassName 
        FROM
            class_first_sec cfs
            LEFT JOIN classificationfirst cf ON cfs.classFirstId = cf.id
            LEFT JOIN classificationsec cs ON cfs.classSecId = cs.id order by desc cfs.firstClassName
code: "ER_PARSE_ERROR"
errno: 1064
sqlMessage: "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'desc cfs.firstClassName' at line 4"
sqlState: "42000"
index: 0
sql: "\n        SELECT\n            cfs.*,\n            cf.NAME firstClassName,\n            cs.NAME secClassName \n        FROM\n            class_first_sec cfs\n            LEFT JOIN classificationfirst cf ON cfs.classFirstId = cf.id\n            LEFT JOIN classificationsec cs ON cfs.classSecId = cs.id order by desc cfs.firstClassName"
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true"}
name: "ER_PARSE_ERRORError"
pid: 19068
hostname: LYU129PRI2

2018-10-26 17:00:42,488 ERROR 27636 [-/::1/-/144ms GET /getClass] nodejs.ER_PARSE_ERRORError: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'desc cfs.firstClassName' at line 4
    at Query.Sequence._packetToError (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Protocol.js:278:23)
    at Parser.write (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Connection.js:91:28)
    at Socket.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Connection.js:502:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    --------------------
    at Pool.query (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\lib\client.js:34:20)
    at RDSClient.proto.query (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\linge l yu\Desktop\egg-example\node_modules\co\index.js:65:19)
    at C:\Users\linge l yu\Desktop\egg-example\node_modules\co\index.js:54:5
    sql: 
        SELECT
            cfs.*,
            cf.NAME firstClassName,
            cs.NAME secClassName 
        FROM
            class_first_sec cfs
            LEFT JOIN classificationfirst cf ON cfs.classFirstId = cf.id
            LEFT JOIN classificationsec cs ON cfs.classSecId = cs.id order by desc cfs.firstClassName
code: "ER_PARSE_ERROR"
errno: 1064
sqlMessage: "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'desc cfs.firstClassName' at line 4"
sqlState: "42000"
index: 0
sql: "\n        SELECT\n            cfs.*,\n            cf.NAME firstClassName,\n            cs.NAME secClassName \n        FROM\n            class_first_sec cfs\n            LEFT JOIN classificationfirst cf ON cfs.classFirstId = cf.id\n            LEFT JOIN classificationsec cs ON cfs.classSecId = cs.id order by desc cfs.firstClassName"
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true"}
name: "ER_PARSE_ERRORError"
pid: 27636
hostname: LYU129PRI2

2018-10-26 17:02:00,492 ERROR 36736 [-/::1/-/185ms GET /getClass] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'cfs.firstClassName' in 'order clause'
    at Query.Sequence._packetToError (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Protocol.js:278:23)
    at Parser.write (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Connection.js:91:28)
    at Socket.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Connection.js:502:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    --------------------
    at Pool.query (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\lib\client.js:34:20)
    at RDSClient.proto.query (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\linge l yu\Desktop\egg-example\node_modules\co\index.js:65:19)
    at C:\Users\linge l yu\Desktop\egg-example\node_modules\co\index.js:54:5
    sql: 
        SELECT
            cfs.*,
            cf.NAME firstClassName,
            cs.NAME secClassName 
        FROM
            class_first_sec cfs
            LEFT JOIN classificationfirst cf ON cfs.classFirstId = cf.id
            LEFT JOIN classificationsec cs ON cfs.classSecId = cs.id order by cfs.firstClassName
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'cfs.firstClassName' in 'order clause'"
sqlState: "42S22"
index: 0
sql: "\n        SELECT\n            cfs.*,\n            cf.NAME firstClassName,\n            cs.NAME secClassName \n        FROM\n            class_first_sec cfs\n            LEFT JOIN classificationfirst cf ON cfs.classFirstId = cf.id\n            LEFT JOIN classificationsec cs ON cfs.classSecId = cs.id order by cfs.firstClassName"
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true"}
name: "ER_BAD_FIELD_ERRORError"
pid: 36736
hostname: LYU129PRI2

2018-10-26 17:02:49,064 ERROR 34216 [-/::1/-/147ms GET /getClass] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'cfs.firstClassName' in 'order clause'
    at Query.Sequence._packetToError (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Protocol.js:278:23)
    at Parser.write (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Connection.js:91:28)
    at Socket.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Connection.js:502:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    --------------------
    at Pool.query (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\lib\client.js:34:20)
    at RDSClient.proto.query (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\linge l yu\Desktop\egg-example\node_modules\co\index.js:65:19)
    at C:\Users\linge l yu\Desktop\egg-example\node_modules\co\index.js:54:5
    sql: 
        SELECT
            cfs.*,
            cf.NAME firstClassName,
            cs.NAME secClassName 
        FROM
            class_first_sec cfs
            LEFT JOIN classificationfirst cf ON cfs.classFirstId = cf.id
            LEFT JOIN classificationsec cs ON cfs.classSecId = cs.id order by cfs.firstClassName
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'cfs.firstClassName' in 'order clause'"
sqlState: "42S22"
index: 0
sql: "\n        SELECT\n            cfs.*,\n            cf.NAME firstClassName,\n            cs.NAME secClassName \n        FROM\n            class_first_sec cfs\n            LEFT JOIN classificationfirst cf ON cfs.classFirstId = cf.id\n            LEFT JOIN classificationsec cs ON cfs.classSecId = cs.id order by cfs.firstClassName"
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true"}
name: "ER_BAD_FIELD_ERRORError"
pid: 34216
hostname: LYU129PRI2

2018-10-26 17:04:55,154 ERROR 29712 [-/::1/-/124ms GET /getClass] nodejs.ER_PARSE_ERRORError: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'by cfs.firstClassName' at line 4
    at Query.Sequence._packetToError (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Protocol.js:278:23)
    at Parser.write (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Connection.js:91:28)
    at Socket.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Connection.js:502:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    --------------------
    at Pool.query (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\lib\client.js:34:20)
    at RDSClient.proto.query (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\linge l yu\Desktop\egg-example\node_modules\co\index.js:65:19)
    at C:\Users\linge l yu\Desktop\egg-example\node_modules\co\index.js:54:5
    sql: 
        SELECT
            cfs.*,
            cf.NAME firstClassName,
            cs.NAME secClassName 
        FROM
            class_first_sec cfs
            LEFT JOIN classificationfirst cf ON cfs.classFirstId = cf.id
            LEFT JOIN classificationsec cs ON cfs.classSecId = cs.idorder by cfs.firstClassName
code: "ER_PARSE_ERROR"
errno: 1064
sqlMessage: "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'by cfs.firstClassName' at line 4"
sqlState: "42000"
index: 0
sql: "\n        SELECT\n            cfs.*,\n            cf.NAME firstClassName,\n            cs.NAME secClassName \n        FROM\n            class_first_sec cfs\n            LEFT JOIN classificationfirst cf ON cfs.classFirstId = cf.id\n            LEFT JOIN classificationsec cs ON cfs.classSecId = cs.idorder by cfs.firstClassName"
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true"}
name: "ER_PARSE_ERRORError"
pid: 29712
hostname: LYU129PRI2

2018-10-26 17:05:20,000 ERROR 36584 [-/::1/-/108ms GET /getClass] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'cfs.firstClassName' in 'order clause'
    at Query.Sequence._packetToError (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Protocol.js:278:23)
    at Parser.write (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Connection.js:91:28)
    at Socket.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Connection.js:502:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    --------------------
    at Pool.query (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\lib\client.js:34:20)
    at RDSClient.proto.query (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\linge l yu\Desktop\egg-example\node_modules\co\index.js:65:19)
    at C:\Users\linge l yu\Desktop\egg-example\node_modules\co\index.js:54:5
    sql: 
        SELECT
            cfs.*,
            cf.NAME firstClassName,
            cs.NAME secClassName 
        FROM
            class_first_sec cfs
            LEFT JOIN classificationfirst cf ON cfs.classFirstId = cf.id
            LEFT JOIN classificationsec cs ON cfs.classSecId = cs.id order by cfs.firstClassName
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'cfs.firstClassName' in 'order clause'"
sqlState: "42S22"
index: 0
sql: "\n        SELECT\n            cfs.*,\n            cf.NAME firstClassName,\n            cs.NAME secClassName \n        FROM\n            class_first_sec cfs\n            LEFT JOIN classificationfirst cf ON cfs.classFirstId = cf.id\n            LEFT JOIN classificationsec cs ON cfs.classSecId = cs.id order by cfs.firstClassName"
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true"}
name: "ER_BAD_FIELD_ERRORError"
pid: 36584
hostname: LYU129PRI2

2018-10-26 17:06:58,538 ERROR 34112 [-/::1/-/67ms GET /getClass] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'cfs.firstClassName' in 'order clause'
    at Query.Sequence._packetToError (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Protocol.js:278:23)
    at Parser.write (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Connection.js:91:28)
    at Socket.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Connection.js:502:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    --------------------
    at Pool.query (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\lib\client.js:34:20)
    at RDSClient.proto.query (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\linge l yu\Desktop\egg-example\node_modules\co\index.js:65:19)
    at C:\Users\linge l yu\Desktop\egg-example\node_modules\co\index.js:54:5
    sql: 
        SELECT
            cfs.*,
            cf.NAME firstClassName,
            cs.NAME secClassName 
        FROM
            class_first_sec cfs
            LEFT JOIN classificationfirst cf ON cfs.classFirstId = cf.id
            LEFT JOIN classificationsec cs ON cfs.classSecId = cs.id order by cfs.firstClassName
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'cfs.firstClassName' in 'order clause'"
sqlState: "42S22"
index: 0
sql: "\n        SELECT\n            cfs.*,\n            cf.NAME firstClassName,\n            cs.NAME secClassName \n        FROM\n            class_first_sec cfs\n            LEFT JOIN classificationfirst cf ON cfs.classFirstId = cf.id\n            LEFT JOIN classificationsec cs ON cfs.classSecId = cs.id order by cfs.firstClassName"
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true"}
name: "ER_BAD_FIELD_ERRORError"
pid: 34112
hostname: LYU129PRI2

2018-10-26 17:07:40,113 ERROR 36444 [-/::1/-/111ms GET /getClass] nodejs.ER_PARSE_ERRORError: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'desc cf.name' at line 4
    at Query.Sequence._packetToError (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Protocol.js:278:23)
    at Parser.write (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Connection.js:91:28)
    at Socket.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Connection.js:502:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    --------------------
    at Pool.query (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\lib\client.js:34:20)
    at RDSClient.proto.query (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\linge l yu\Desktop\egg-example\node_modules\co\index.js:65:19)
    at C:\Users\linge l yu\Desktop\egg-example\node_modules\co\index.js:54:5
    sql: 
        SELECT
            cfs.*,
            cf.NAME firstClassName,
            cs.NAME secClassName 
        FROM
            class_first_sec cfs
            LEFT JOIN classificationfirst cf ON cfs.classFirstId = cf.id
            LEFT JOIN classificationsec cs ON cfs.classSecId = cs.id order by desc cf.name
code: "ER_PARSE_ERROR"
errno: 1064
sqlMessage: "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'desc cf.name' at line 4"
sqlState: "42000"
index: 0
sql: "\n        SELECT\n            cfs.*,\n            cf.NAME firstClassName,\n            cs.NAME secClassName \n        FROM\n            class_first_sec cfs\n            LEFT JOIN classificationfirst cf ON cfs.classFirstId = cf.id\n            LEFT JOIN classificationsec cs ON cfs.classSecId = cs.id order by desc cf.name"
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true"}
name: "ER_PARSE_ERRORError"
pid: 36444
hostname: LYU129PRI2

2018-10-26 17:26:27,690 ERROR 564 nodejs.unhandledExceptionError: read ECONNRESET
    at _errnoException (util.js:992:11)
    at TCP.onread (net.js:618:25)
code: "ECONNRESET"
errno: "ECONNRESET"
syscall: "read"
name: "unhandledExceptionError"
pid: 564
hostname: LYU129PRI2

2018-10-26 17:27:45,133 ERROR 25888 [-/::1/-/3ms GET /findLists] nodejs.TypeError: Cannot use 'in' operator to search for 'typeCast' in null
    at Pool.query (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Pool.js:193:47)
    at C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\lib\client.js:34:20)
    at RDSClient.proto.query (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\linge l yu\Desktop\egg-example\node_modules\co\index.js:65:19)
    at C:\Users\linge l yu\Desktop\egg-example\node_modules\co\index.js:54:5
    sql: null
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true"}
pid: 25888
hostname: LYU129PRI2

2018-10-26 17:37:19,507 ERROR 32704 [-/::1/-/6ms GET /findLists?pageNum=1&pageSize=10&search=12] nodejs.ER_PARSE_ERRORError: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'like productName = '%12%'' at line 1
    at Query.Sequence._packetToError (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Protocol.js:278:23)
    at Parser.write (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Connection.js:91:28)
    at Socket.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Connection.js:502:10)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    --------------------
    at Pool.query (C:\Users\linge l yu\Desktop\egg-example\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\lib\client.js:34:20)
    at RDSClient.proto.query (C:\Users\linge l yu\Desktop\egg-example\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\linge l yu\Desktop\egg-example\node_modules\co\index.js:65:19)
    at C:\Users\linge l yu\Desktop\egg-example\node_modules\co\index.js:54:5
    sql: select * from product limit 0, 10 like productName = '%12%'
code: "ER_PARSE_ERROR"
errno: 1064
sqlMessage: "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'like productName = '%12%'' at line 1"
sqlState: "42000"
index: 0
sql: "select * from product limit 0, 10 like productName = '%12%'"
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true"}
name: "ER_PARSE_ERRORError"
pid: 32704
hostname: LYU129PRI2

2018-10-29 09:58:44,146 ERROR 564 nodejs.ClusterClientNoResponseError: client no response in 231657058ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (C:\Users\linge l yu\Desktop\egg-example\node_modules\cluster-client\lib\leader.js:74:23)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
name: "ClusterClientNoResponseError"
pid: 564
hostname: LYU129PRI2

